src/main/resources/XWiki/Macros/Team.xml

```diff
@@ -0,0 +1,1407 @@
+    obj3.id = tagprop.id.id and
+    tagprop.id.name = 'tags' and
+    list = '${tag}'
+  ")
+#end
+#set($sql = "SELECT doc.fullName FROM XWikiDocument doc,
+    BaseObject as obj
+    #if($requireExternalAuth), BaseObject as obj2 #end,
+    IntegerProperty as prop
+    $tagselect
+  WHERE
+    doc.fullName = obj.name and
+    obj.className = 'XWiki.XWikiUsers'
+    #if($requireExternalAuth) and
+      doc.fullName = obj2.name and (
+        obj2.className = 'XWiki.LDAPProfileClass' or
+        obj2.className = 'XWiki.OIDC.UserClass'
+      )
+    #end and
+    obj.id = prop.id.id and
+    prop.id.name = 'active' and
+    prop.value = 1
+    $tagwhere
+  ORDER BY doc.name
+")
+#set($list = $services.query.hql($sql).execute())
+{{html wiki=false clean=false}}
+&lt;div class="xwikiteam #if($showUsernames)xwikiteam-usernames-displayed #else xwikiteam-usernames-hidden #end"&gt;
+  #if ($list.size() &gt; 0)
+    &lt;ul class="xwikiteam-ul"&gt;
+      #foreach($user in $list)
+        #avatar("xwiki:${user}" $size $disableLetterAvatars $letterAvatarBgColor $letterAvatarFontColor)
+      #end
+      #if(!$disableTools)
+        &lt;li class="xwikiteam-tools-link" hidden="hidden"&gt;
+          &lt;a href="#" role="button" title="${services.localization.render('rendering.macro.team.content.options')}"&gt;
+            &lt;span class="fa fa-wrench"&gt;&lt;/span&gt;
+            &lt;span class="sr-only"&gt;${services.localization.render('rendering.macro.team.content.options')}&lt;/span&gt;
+          &lt;/a&gt;
+        &lt;/li&gt;
+      #end
+    &lt;/ul&gt;
+    &lt;div class="xwikiteam-tools" hidden="hidden"&gt;
+      &lt;input class="xwikiteam-filter" type="text" placeholder="${services.localization.render('rendering.macro.team.content.filter')}" /&gt;
+      &lt;label&gt;
+        &lt;input
+          class="xwikiteam-show-username-checkbox"
+          type="checkbox"
+          #if($displayUsernames)checked="checked"#end
+        /&gt;
+        ${services.localization.render('rendering.macro.team.content.showUsernames')}
+      &lt;/label&gt;
+    &lt;/div&gt;
+  #else
+    &lt;p&gt;${services.localization.render('rendering.macro.team.content.noUsers')}&lt;/p&gt;
+  #end
+&lt;/div&gt;
+{{/html}}
+{{/velocity}}</code>
+    </property>
+    <property>
+      <contentDescription></contentDescription>
+    </property>
+    <property>
+      <contentJavaType></contentJavaType>
+    </property>
+    <property>
+      <contentType>No content</contentType>
+    </property>
+    <property>
+      <defaultCategory>content</defaultCategory>
+    </property>
+    <property>
+      <description>This macro displays the team members which have a specific tag</description>
+    </property>
+    <property>
+      <id>team</id>
+    </property>
+    <property>
+      <name>XWiki Team Image Display</name>
+    </property>
+    <property>
+      <priority/>
+    </property>
+    <property>
+      <supportsInlineMode>1</supportsInlineMode>
```

Comment:
I see it always generates DIVs so I don't see how it can be used inline, e.g. inside a paragraph.

Reply:
That's something I didn't change rather than a conscious decision, I'll disable this.
```suggestion
      <supportsInlineMode>0</supportsInlineMode>
```